tensorflow/java/src/main/java/org/tensorflow/NativeLibrary.java

```diff
@@ -139,9 +133,9 @@ private static boolean resourceExists(String baseName) {
 
   private static String getVersionedLibraryName(String libFilename) {
     // If the resource exists as an unversioned file, return that.
-    if (resourceExists(libFilename)) {
-      return libFilename;
-    }
+    // if (resourceExists(libFilename)) {
```

Comment:
Comment-out left over from debugging?

Reply:
正在使用发音
This is the original code, but because there is libtensorfow_framework.so in the jar package, which causes the later logic to fail, I temporarily commented out the code.

Reply:
I'm confused then. If the unversioned shared object exists, why isn't it used? This logic did work before we added version numbers to shared objects.

Reply:
There are libtensorflow_framework.so.so and libtensorflow_framework.so.1. libtensorflow_jni.so seems to depend on libtensorflow_framework.so.1, but now the logic detects that libtensorflow_framework.so is over. Does he seem to have a problem?

Reply:
Thanks for the explanation. Since we'd fall back to the unversioned lib anyway I think we can just delete this snippet rather than commenting out.

Reply:
I think that's a good suggestion. Now I'll delete the logic of judgment.